### PR TITLE
fix: kic 2.8 enabled CombinedRoutes by default

### PR DESCRIPTION
**Pull request by randmonkey**
### Description

KIC 2.8.x has enabled feature gate `CombinedRoutes` by default, so docs should be updated.

### Testing instructions

Netlify link: 
https://deploy-preview-5139--kongdocs.netlify.app/kubernetes-ingress-controller/2.8.x/references/feature-gates/

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Review by cloudjumpercat — APPROVED** [review]
LGTM! I'll wait until @rainest reviews and then I can merge this in

**Review by rainest — APPROVED** [review]

**Pull request merged**
